README.md

```diff
@@ -69,10 +65,24 @@ yarn start
 
 Open `localhost:8080` on your browser
 
-# Projects based on this validator
+## Projects based on this validator
+
+[transport.data.gouv.fr GBFS validator tool](https://transport.data.gouv.fr/validation?type=gbfs) - Tool displaying interactive geofencing, station, and vehicle maps, the validation results, and metadata of GBFS feeds.
 
-[transport.data.gouv.fr GBFS validator tool](https://transport.data.gouv.fr/validation?type=gbfs) - Tool displaying interactive geofencing, station, and vehicle maps, the validation results, and metadata of GBFS feeds.  
+## Contributors
 
-# Acknowledgements
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->
+
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome! Please check out our [Contribution guidelines](/CONTRIBUTING.md) for details.
```

Comment:
It could be useful to link to the emoji key there https://allcontributors.org/docs/en/emoji-key

Reply:
Done!